tests/chainer_tests/functions_tests/loss_tests/test_mean_squared_error.py

```diff
@@ -46,9 +58,11 @@ def test_forward_gpu(self):
         self.check_forward(cuda.to_gpu(self.x0), cuda.to_gpu(self.x1))
 
     def check_backward(self, x0_data, x1_data):
+        atol = self.backward_tols['atol']
+        rtol = self.backward_tols['rtol']
         gradient_check.check_backward(
             functions.mean_squared_error,
-            (x0_data, x1_data), None, eps=1e-2)
+            (x0_data, x1_data), None, eps=1e-2, atol=atol, rtol=rtol)
```

Comment:
Why don't you use `**self.backward_tols`?

Reply:
Yes, it's enough!